set-web/src/main/java/eu/europeana/set/web/search/UserSetQueryBuilder.java

```diff
@@ -70,10 +70,10 @@ private UserSetQuery buildSearchQuery(Map<String, String> searchCriteria, String
     private void addTitleLangCriterion(Map<String, String> searchCriteria, UserSetQuery searchQuery)
         throws ParamValidationException {
     if (searchCriteria.containsKey(WebUserSetFields.LANG)) {
-        String lang = searchCriteria.get(WebUserSetFields.LANG);
-        if (!"en".equals(lang)) {
+        String lang = searchCriteria.get(WebUserSetFields.LANG).toLowerCase();
+        if (! UserSetUtils.is2Letter(lang)) {
```

Comment:
please replace the implementation for validating the value of the language against the real ISO Language codes. 
Available in the Locale.getISOLanguages()

Reply:
please use the same solution from the entity management https://github.com/europeana/entity-management/blob/cb34d4dd448a4328d0cc57d2ee3dfb291f352a24/entity-management-definitions/src/main/java/eu/europeana/entitymanagement/normalization/EntityFieldsCleaner.java#L41

Reply:
A new class LanguageUtils can be implemented in the api coommons in this package: eu.europeana.api.commons.definitions.utils

Reply:
Improved.